ramls/loan.json

```diff
@@ -103,6 +103,11 @@
       "type": "string",
       "format": "date-time"
     },
+    "agedToLostDate": {
+      "description": "Date and time the item was aged to lost for this loan",
+      "type": "string",
+      "format": "date-time"
+    },
```

Comment:
What is this property going to be used for? Is it specifically related to deciding whether to charge a fee or fine to a patron?

Reply:
Just for display purpose only, AFAIK. It can be used to calculate an overdue fine, I think.